### PR TITLE
Upgrade markdeep to v1.16 to fix the subheading issue

### PR DESCRIPTION
**Pull request by painfulexistence**
There seemed to an issue of subheading index not correctly reset in Markdeep, which is mentioned here (https://github.com/google/filament/issues/3525) and can be observed from the below screenshot.

Screenshot 2024-07-07 at 11 45 52 AM

I upgraded Markdeep to v1.16 (https://github.com/morgan3d/markdeep/releases/tag/v01.16) to fix the issue.

This PR also makes sure that the same JS (markdeep.min.js) is always used across pages.

### CODE REVIEW

File: scripts/LICENSE
Comment (nickclark2016) [inline scripts/LICENSE:1]: Can we rename this to markdeep.LICENSE (or something like that) so we know what this license file is for just at a glance?

Comment (painfulexistence) [reply scripts/LICENSE:1]: Sure! But how about moving LICENSE and markdeep.min.js into a folder, I.e. scripts/markdeep?

Comment (nickclark2016) [reply scripts/LICENSE:1]: Seems reasonable go me

Comment (painfulexistence) [reply scripts/LICENSE:1]: Done!

**Review by nickclark2016 — APPROVED** [review]

**Pull request merged**